samza-kv-couchbase/src/main/java/org/apache/samza/table/remote/couchbase/CouchbaseEnvironmentConfigs.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.samza.table.remote.couchbase;
+
+import java.io.Serializable;
+
+
+class CouchbaseEnvironmentConfigs implements Serializable {
```

Comment:
Why creating this class other than use DefaultCouchbaseEnvironment.Builder  directly?

Reply:
Good question. I really wish to use DefaultCouchbaseEnvironment.Builder. But the problem is that class is not serializable. So we have to create this serializable CouchbaseEnvironmentConfigs to keep the configs.